projects/kit/components/input-count/input-count.component.ts

```diff
@@ -118,7 +126,7 @@ export class TuiInputCountComponent
 
     @HostBinding('class._has-buttons')
     get hasButtons(): boolean {
```

Comment:
Unnecessary getter, let's revert what CSS does and bind it to input directly

Reply:
Done